Arista/arista-package.sh

```diff
@@ -92,6 +92,8 @@ mkdir -p $TMP_CONFIG_DIR/lib/systemd/system
 cp $CONFIG_FILES_DIR/telegraf.service $TMP_CONFIG_DIR/lib/systemd/system/telegraf.service
 mkdir -p $TMP_CONFIG_DIR/etc/telegraf
 mkdir -p $TMP_CONFIG_DIR/etc/telegraf/telegraf.d
+mkdir -p $TMP_CONFIG_DIR/usr/lib/telegraf/scripts
```

Comment:
since we are changing the configuration, we should increase the config file version to ensure an update happens. For some reason I can't put a comment on unchanged lines.

CONFIG_FILES_VER needs to be bumped to 1.5